obsproject/obs-studio
# Windows: Taskbar icon & colour

### Description
This adds a taskbar colour & status icon on Windows 8 and above.
![image](https://user-images.githubusercontent.com/941350/75540192-a687aa80-5a6f-11ea-85ac-a406586dcf10.png) ![image](https://user-images.githubusercontent.com/941350/75540198-a9829b00-5a6f-11ea-9f64-551feeb00ce8.png)
**No output**: normal. **Live output**: green. **Paused recording**: yellow.

Not actually sure if the overlay icons should be configurable by the theme. **Feedback welcome in this area.**

### Motivation and Context
Users often mention that live/recording/paused status is not visually obvious enough. Colour at a glance, along with an icon, should help.

### How Has This Been Tested?
* Launch OBS on Windows 8 or above
* Start Recording. Notice the colour of the taskbar button changes to green, and 🔴 appears on top.
* Pause the recording. Notice the taskbar button changes to yellow, and the 🔴 changes to a yellow paused icon.


### Types of changes
 - New feature (non-breaking change which adds functionality)

### Checklist:
